docs/docs/20-quickstart.md

```diff
@@ -508,52 +510,28 @@ the previous section.
     EOF
     ```
     
-  :::info
-  Kargo uses [semver](https://github.com/masterminds/semver#checking-version-constraints) to handle semantic versioning constraints.
-  :::
-
-1. Use the CLI to view our `Warehouse` resource:
-
-    ```shell
-    kargo get warehouses --project kargo-demo
-    ```
+      :::info
+      Kargo uses [semver](https://github.com/masterminds/semver#checking-version-constraints) to handle semantic versioning constraints.
+      :::
 
-    Sample output:
+1. After applying the above configuration, you can view the pipeline you just created using the [Kargo Dashboard](https://localhost:31444/).
 
-    ```shell
-    NAME         SHARD   AGE
-    kargo-demo           13s
-    ```
+   1. Choose your `Project`, <Hlt>kargo-demo</Hlt>, from the list of available `Project`s
+   1. You can find the pipeline you've created in the center of your Project screen.
+   1. The pipeline consists of connected rectangles representing:
+      1. <Hlt>Subscription</Hlt>: Has information about the container image repository that the `Warehouse` is subscribed to.
+      1. <Hlt>kargo-kemo</Hlt>: This represents your `Warehouse`. It will automatically subscribe to the image and should produce `Freight` shortly.
+      1. <Hlt>test</Hlt>, <Hlt>uat</Hlt>, and <Hlt>prod</Hlt>: These are the deployment `Stage` resources.
```

Comment:
If this is someone's first time through the quickstart (and I think that's a safe thing to assume), they probably need to log into the dashboard.

This section also reads oddly, as it starts with imperatives ("do this") and then jumps immediately into information the user _doesn't need to act on_, but they're not really presented any differently than the things they do. You don't want someone wondering, "Am I supposed to do something here?"

```suggestion
   1. Log in using the password `admin`.
   
       This will take you to the home page of the dashboard.
       
       <Good place for a screen shot>
   
   1. Select your Project, <Hlt>kargo-demo</Hlt>, from the list.
   
       Here you can see a detailed overview of the Project we created using the CLI, including:
       
           * An interactive, visual representation of your pipeline:
               * A container image repository
               * A Warehouse that discovers new images as they are pushed to the repository.
               * Three Stages representing distinct instances of our demo application

           <Good place for a screen shot. Maybe highlight the pipeline.>

           * And interactive Freight timeline:
               * This displays collections of artifacts that have been discovered by the Warehouse.
               * Collections (Freight) are ordered chronologically, with newer Freight on the left and older Freight on the right.
                     
           <Good place for a screen shot. Maybe highlight the Freight timeline.>
```

The above suggestion may still need some formatting fixes and such.